index.html

```diff
@@ -17,14 +17,22 @@
             <a href="#"><h2>Bilal</h2></a>
         </div>
         <nav class="Menuicon">
-            <span class="material-symbols-outlined">
+            <span id="navicon" class="material-symbols-outlined">
                 menu
                 </span>
         </nav>
-        <ul class="menu">
-            <li><a class="menu_item" href="#portfolio">portfolio</a></li>
-            <li><a class="menu_item" href="#about">About</a></li>
-            <li><a class="menu_item" href="#contact">Contact</a></li>
+        <ul id="menu" class="menu">
+            <a href="#" id="cross" class="Menuiconcross"
+					><span
+						id="crossIcon"
+						style="color: white"
+						class="material-symbols-outlined"
+					>
+						close
+					</span></a>
```

Comment:
- [ ] Please note that the close icon is on the right in the figma design. Kindly make this change to match the Figma.
![menu](https://github.com/BilalLiaquat7/Portfolio/assets/99656687/2ea240b8-04d3-4e1c-aa58-39e4ee3ff36c)